### PR TITLE
NO-JIRA Uses script to build dependencies

### PR DESCRIPTION
**Pull request by davividal**

### CODE REVIEW

File: .cirrus/tasks.yml [hunk 1]
```diff
@@ -156,14 +153,12 @@ chart_static_compatibility_test_task:
     memory: 1Gb
   <<: *CLONE_SCRIPT_TEMPLATE
   script:
-    - helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
-    - helm repo add bitnami-pre2022 https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
+    - cd "${CIRRUS_WORKING_DIR}" && ./.cirrus/build_chart_dependencies.sh charts/sonarqube
```
Comment (jCOTINEAU) [inline .cirrus/tasks.yml:156]: cd "${CIRRUS_WORKING_DIR} this is not needed as the script start in CIRRUS_WORKING_DIR

File: .cirrus/tasks.yml [hunk 2]
```diff
@@ -174,13 +169,9 @@ chart_fixture_test_task:
...
-    - helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
-    - helm repo add bitnami-pre2022 https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
-    - cd ${CIRRUS_WORKING_DIR}/charts/sonarqube
-    - helm dependency build
-    - cd ${CIRRUS_WORKING_DIR}/charts/sonarqube-dce
-    - helm dependency build
-    - cd ../..
+    - cd "${CIRRUS_WORKING_DIR}" && ./.cirrus/build_chart_dependencies.sh charts/sonarqube
+    - cd "${CIRRUS_WORKING_DIR}" && ./.cirrus/build_chart_dependencies.sh charts/sonarqube-dce
+    - cd ${CIRRUS_WORKING_DIR}
```
Comment (jCOTINEAU) [inline .cirrus/tasks.yml:174]: Same here all CD could be removed I guess

Comment (davividal) [reply .cirrus/tasks.yml:174]: Nice catch! Thanks

File: .cirrus/tasks.yml [hunk 3]
```diff
@@ -156,14 +153,12 @@ chart_static_compatibility_test_task:
...
+    - cd "${CIRRUS_WORKING_DIR}" && ./.cirrus/build_chart_dependencies.sh charts/sonarqube
     - cd ${CIRRUS_WORKING_DIR}/charts/sonarqube
-    - helm dependency build
-    - ../../.cirrus/unit_helm_compatibility_test.sh
+    - "${CIRRUS_WORKING_DIR}"/.cirrus/unit_helm_compatibility_test.sh
     - cd ${CIRRUS_WORKING_DIR}/charts/sonarqube-dce
-    - helm dependency build
-    - ../../.cirrus/unit_helm_compatibility_test.sh    
+    - cd "${CIRRUS_WORKING_DIR}" && ./.cirrus/build_chart_dependencies.sh charts/sonarqube-dce
+    - "${CIRRUS_WORKING_DIR}"/.cirrus/unit_helm_compatibility_test.sh
```
[the remark this answers is not in the document]
Comment (davividal) [reply .cirrus/tasks.yml]: Done!

**Comment by carminevassallo** [comment]
@davividal can you please rebase so that we can check the latest CI updates against your PR?

**Review by carminevassallo — APPROVED** [review]
LGTM! Great Refactoring 👏

**Pull request merged**